.circleci/config.yml

```diff
@@ -159,6 +159,7 @@ jobs:
           command: npm run system-test
           environment:
             GOOGLE_APPLICATION_CREDENTIALS: .circleci/key.json
+            GCLOUD_PROJECT: long-door-651
```

Comment:
This is just going to get reverted when synthtool runs next, isn't it?  Can't we just do this for everyone in the templates?  Also, is this covered in kokoro?